src/d3d12/D3D12.h

```diff
@@ -15,6 +15,8 @@ struct D3D12
 {
     inline static const uint32_t g_numDownlevelBackbuffersRequired = 3; // Windows 7 only: number of buffers needed before we start rendering
 
+    inline static const SIZE g_designResolution = { 1920, 1080 };
```

Comment:
Hm, how would you structure it that way? I used `SIZE` because it's what the actual resolution value uses - so this is literally `1920x1080` (and I guess you could provide multiple to select from or allow the user to supply their own reference resolution.)

In an enum the x and y would be divorced from one another? So my typing sense would make it something like

```c++
const enum Resolutions {
  DefaultDesignResX = 1920,
  DefaultDesignResY = 1080
}
```
Am I missing a different way to use enums here?